docs/cody/clients/install-cli.mdx

```diff
@@ -0,0 +1,212 @@
+# Install Cody CLI
+
+<p className="subtitle">Learn how to install the <code>cody</code> command-line tool.</p>
+
+<Callout type="note" title="Experimental">
+Cody CLI support is in the experimental stage.
+</Callout>
+
+The Cody extension by Sourcegraph enhances your coding experience in your IDE by providing intelligent code suggestions, context-aware completions, and advanced code analysis. This guide will walk you through the steps to install and set up Cody within your JetBrains environment.
```

Comment:
This text feels wrong - I think we should talk about the CLI here.

Reply:
good catch, I copy-pasted the JB page. Wrote instead

> Cody CLI is the same technology that powers the Cody IDE plugins but available from the command-line.
> Use Cody CLI for ad-hoc exploration in your terminal or as part of scripts to automate your workflows.